packages/components/Link/src/Link.settings.ts

```diff
@@ -7,8 +7,8 @@ export const settings: IComposeSettings<ILinkType> = [
   {
     tokens: {
       fontFamily: 'primary',
-      fontSize: 'medium',
-      fontWeight: 'medium',
+      fontSize: 'secondary',
+      fontWeight: 'regular',
       color: 'link'
```

Comment:
Should this use a variant?

Reply:
Probably. Updated this to use fontVariant in recent commits.